mappings/net/minecraft/data/client/model/VariantSettings.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/class_4936 net/minecraft/data/client/model/VariantSettings
+	FIELD field_22885 X Lnet/minecraft/class_4938;
+	FIELD field_22886 Y Lnet/minecraft/class_4938;
+	FIELD field_22887 MODEL Lnet/minecraft/class_4938;
+	FIELD field_22888 UVLOCK Lnet/minecraft/class_4938;
```

Comment:
`UV_LOCK`

Reply:
https://minecraft.gamepedia.com/Model#Block_states
It's not `uvLock` but `uvlock`, as seen at 
![image](https://user-images.githubusercontent.com/7806504/75945312-72d9c580-5e5f-11ea-82ff-955f843c26e0.png)